tests/test_md.py

```diff
@@ -504,16 +504,12 @@ def test_get_movers():
     friction = 0.0
     seed = 2022
 
-    unbound_potentials, sys_params, masses, coords, box = get_solvent_phase_system(mol, ff, 0.0, minimize_energy=True)
+    unbound_potentials, sys_params, masses, coords, box = get_solvent_phase_system(mol, ff, 0.0, minimize_energy=False)
```

Comment:
Speed up the test, it only verifies that the movers can be retrieved, no reason to minimize or execute the potentials.